apps/omg_db/test/omg_db/db_test.exs

```diff
@@ -63,6 +64,97 @@ defmodule OMG.DBTest do
     checks.(pid)
   end
 
+  test "block hashes return the correct range", %{db_dir: _dir, db_pid: pid} do
+    :ok =
+      DB.multi_update(
+        [
+          {:put, :block, %{hash: "xyz", number: 1}},
+          {:put, :block, %{hash: "vxyz", number: 2}},
+          {:put, :block, %{hash: "wvxyz", number: 3}}
+        ],
+        pid
+      )
+
+    {:ok, ["xyz", "vxyz", "wvxyz"]} = OMG.DB.block_hashes([1, 2, 3], pid)
+  end
+
+  test "if multi reading exit infos returns writen results", %{db_dir: _dir, db_pid: pid} do
+    db_writes = create_write(:exit_info, pid)
+    {:ok, exits} = DB.exit_infos(pid)
+    [] = exits -- db_writes
+  end
+
+  test "if multi reading utxos returns writen results", %{db_dir: _dir, db_pid: pid} do
+    db_writes = create_write(:utxo, pid)
+    {:ok, utxos} = DB.utxos(pid)
+    [] = utxos -- db_writes
+  end
+
+  test "if multi reading in flight exit infos returns writen results", %{db_dir: _dir, db_pid: pid} do
+    db_writes = create_write(:in_flight_exit_info, pid)
+    {:ok, in_flight_exits_infos} = DB.in_flight_exits_info(pid)
+    [] = in_flight_exits_infos -- db_writes
+  end
+
+  test "if multi reading competitor infos returns writen results", %{db_dir: _dir, db_pid: pid} do
+    db_writes = create_write(:competitor_info, pid)
+    {:ok, competitors_info} = DB.competitors_info(pid)
+    [] = competitors_info -- db_writes
+  end
+
+  test "if multi reading and writting does not pollute returned values", %{db_dir: _dir, db_pid: pid} do
+    db_writes = create_write(:exit_info, pid)
+    {:ok, exits} = DB.exit_infos(pid)
+    [] = exits -- db_writes
```

Comment:
add `assert` to emphasize what we're actually testing here (and improved error printouts I suppose)